src/Stream/test/StreamBase.Test/Binder/DefaultBinderTypeRegistryTest.cs

```diff
@@ -6,45 +6,51 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Reflection;
+using System.Runtime.InteropServices;
 using Xunit;
 
 namespace Steeltoe.Stream.Binder
 {
-    public class DefaultBinderTypeRegistryTest
+    public class DefaultBinderTypeRegistryTest : AbstractTest
     {
-        [Fact(Skip = "TypeRegistryTests")]
+        [Fact]
         public void LoadAndCheckAssembly_WithValidPath_ReturnsBinderType()
         {
-            var context = new DefaultBinderTypeRegistry.SearchingAssemblyLoadContext();
-            var path = Environment.CurrentDirectory + Path.DirectorySeparatorChar + "Steeltoe.Stream.TestBinder.dll";
-            var result = DefaultBinderTypeRegistry.LoadAndCheckAssembly(context, path);
-            Assert.Equal(path, result.AssemblyPath);
-            Assert.Equal("Steeltoe.Stream.TestBinder.Startup, Steeltoe.Stream.TestBinder, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null", result.ConfigureClass);
+            var binderDir = GetSearchDirectories("TestBinder")[0];
+            var paths = BuildPaths(binderDir);
+
+            var context = new MetadataLoadContext(new PathAssemblyResolver(paths));
+            var binderAssembly = binderDir + Path.DirectorySeparatorChar + "Steeltoe.Stream.TestBinder.dll";
+            var result = DefaultBinderTypeRegistry.LoadAndCheckAssembly(context, binderAssembly);
+            Assert.Equal(binderAssembly, result.AssemblyPath);
+            Assert.Matches(@"Steeltoe.Stream.TestBinder.Startup, Steeltoe.Stream.TestBinder, Version=[\d.]+, Culture=neutral, PublicKeyToken=null", result.ConfigureClass);
```

Comment:
@dtillman is the version critical here or can we do this - Figured this is the easiest to do deal with versions on build server